app/src/organisms/ApplyHistoricOffsets/index.tsx

```diff
@@ -77,54 +77,86 @@ export function ApplyHistoricOffsets(
       {...{ labware, modules, commands }}
     />
   )
+  const noOffsetData = offsetCandidates.length < 1
   return (
     <Flex alignItems={ALIGN_CENTER} justifyContent={JUSTIFY_SPACE_BETWEEN}>
       <CheckboxField
         onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
           setShouldApplyOffsets(e.currentTarget.checked)
         }}
         value={shouldApplyOffsets}
-        disabled={offsetCandidates.length < 1}
-        isIndeterminate={offsetCandidates.length < 1}
+        disabled={noOffsetData}
+        isIndeterminate={noOffsetData}
         label={
           <Flex alignItems={ALIGN_CENTER} gridGap={SPACING.spacing2}>
             <Icon size={SIZE_1} name="reticle" />
-            <StyledText as="p">{t('apply_offset_data')}</StyledText>
+            <StyledText as="p">
+              {t(noOffsetData ? 'no_offset_data' : 'apply_offset_data')}
+            </StyledText>
           </Flex>
         }
       />
       <Link
         onClick={() => setShowOffsetDataModal(true)}
         css={TYPOGRAPHY.linkPSemiBold}
       >
-        {t('view_data')}
+        {t(noOffsetData ? 'learn_more' : 'view_data')}
       </Link>
       {showOffsetDataModal ? (
         <Portal level="top">
           <ModalShell
             maxWidth="40rem"
             header={
               <ModalHeader
-                title={t('stored_offset_data')}
+                title={t(
+                  noOffsetData
+                    ? 'no_offset_data_available'
+                    : 'stored_offset_data'
+                )}
                 onClose={() => setShowOffsetDataModal(false)}
               />
             }
           >
-            <Flex flexDirection={DIRECTION_COLUMN} padding={SPACING.spacing6}>
-              <StyledText as="p">
-                {offsetCandidates.length > 0
-                  ? t('robot_has_offsets_from_previous_runs')
-                  : t('robot_has_no_offsets_from_previous_runs')}
-              </StyledText>
+            <Flex
+              flexDirection={DIRECTION_COLUMN}
+              padding={
+                noOffsetData
+                  ? `${SPACING.spacing4} ${SPACING.spacing6} ${SPACING.spacing6} ${SPACING.spacing6}`
```

Comment:
```suggestion
                  ? `${SPACING.spacing4} ${SPACING.spacing6} ${SPACING.spacing6}`
```
probably remove one since when we put 3 values
`top | left and right | bottom`

Reply:
oops, thanks! I always forget how to shorthand this